cli/src/new.rs

```diff
@@ -81,7 +81,7 @@ impl New {
         let src = self.path.join("src");
         fs::create_dir_all(&src)?;
         fs::write(
-            src.join("lib.rs"),
+            src.join("mod"),
```

Comment:
hmmm why changing it to `mod`?

this command creates a new template zink project from rust library, and the root of the library should be `lib.rs`

Reply:
lol new ide sry